fastlane/Fastfile

```diff
@@ -230,55 +230,6 @@ platform :ios do
   # Release Lanes
   #####################################################################################
 
-  # Executes the code freeze
-  #
-  # - Call start_code_freeze
-  # - Call complete_code_freeze
-  # - Go over PRs with the current milestone and add a comment to move them to the next
-  # - Open a PR targeting trunk
-  # - Send a message to Slack channel confirming the code freeze was finished
-  #
-  # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
-  #
-  desc 'Executes all the code freeze steps'
-  lane :code_freeze do |options|
-    start_code_freeze(options)
-    comment_on_prs(milestone: release_version_current)
-    complete_code_freeze(options)
-  end
```

Comment:
Instead of splitting the implementation of this `code_freeze` lane in separate `start_code_freeze` and `complete-code_freeze` lanes, I moved the whole implementation of the steps in the `code_freeze` lane directly, see [line 270 below](https://github.com/Automattic/pocket-casts-ios/pull/2215/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R270)